hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/CheckWanConsistencyMessageTask.java

```diff
@@ -22,12 +22,16 @@
 import com.hazelcast.client.impl.protocol.task.AbstractCallableMessageTask;
 import com.hazelcast.instance.impl.Node;
 import com.hazelcast.internal.nio.Connection;
+import com.hazelcast.security.permission.ManagementPermission;
 import com.hazelcast.wan.impl.WanReplicationService;
 
 import java.security.Permission;
 import java.util.UUID;
 
 public class CheckWanConsistencyMessageTask extends AbstractCallableMessageTask<RequestParameters> {
+
+    public static final Permission REQUIRED_PERMISSION = new ManagementPermission("wan.checkConsistency");
```

Comment:
Do these need to be `public`?

Reply:
:+1: They don't need to. I'll fix it.
We don't use the constant in other places. And as the tasks are not part of our public API it's better to keep them private.